backend/contract/src/main/kotlin/com/ritense/valtimo/contract/process/ProcessConstants.kt

```diff
@@ -0,0 +1,6 @@
+package com.ritense.valtimo.contract.process
+
+object ProcessConstants {
+    const val OPERATION_CASE_DEFINITION_VERSION_TAG_PREFIX = "CD:"
```

Comment:
This should be operaton instead of operation, right? Same goes for the line below.